omnibus/config/projects/agent.rb

```diff
@@ -296,10 +296,6 @@
   if ENV['SIGN_WINDOWS_DD_WCS']
     BINARIES_TO_SIGN = GO_BINARIES + [
       "#{install_dir}\\bin\\agent\\ddtray.exe",
-      "#{install_dir}\\embedded3\\python.exe",
-      "#{install_dir}\\embedded3\\\\python3.dll",
-      "#{install_dir}\\embedded3\\\\python39.dll",
-      "#{install_dir}\\embedded3\\\\pythonw.exe",
```

Comment:
Could you clarify why those shouldn't be signed anymore? I suppose they are already signed by the python project, but don't we rely on our binaries being signed with our key somewhere else?

Reply:
Ah sorry, I tried to explain that in the description but yes we do not rely on our binaries anymore on Windows. I discussed that part with the windows agent team

Reply:
Oh sorry that's on my I went straight to the changes and ignored the description :facepalm:

Reply:
no worries 🙂